src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2514,11 +2518,19 @@ internal bool CanSelectAll(object parameter)
         public void MakeNewHomeWorkspace(object parameter)
         {
             if (ClearHomeWorkspaceInternal())
-            {
+            {   
                 var t = new DelegateBasedAsyncTask(model.Scheduler, () => model.ResetEngine());
                 model.Scheduler.ScheduleForExecution(t);
 
                 ShowStartPage = false; // Hide start page if there's one.
+
+                var defaultWorkspace = Workspaces.FirstOrDefault();
```

Comment:
Could this be custom node workspace if user opened dyf first? I thought there are pointer already

Reply:
this code was also added in the part in which the user creates a new custom node, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13656/commits/c20979418fa67d8906f184af3810952b71d26641